apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -83,6 +139,29 @@ export const applyBrand = (currentBrand: string) => {
           link: ramp.App6,
           linkHovered: ramp.App7,
           linkPressed: ramp.App8,
+          neutralForeground2BrandHover: ramp.AppPrimary,
+          neutralForeground2BrandPressed: ramp.AppShade10,
+          neutralForeground3BrandHover: ramp.AppPrimary,
+          neutralForeground3BrandPressed: ramp.AppShade10,
+          brandForegroundLink: ramp.AppShade10,
+          brandForegroundLinkHover: ramp.AppShade20,
+          brandForegroundLinkPressed: ramp.AppShade30,
+          compoundBrandForeground1: ramp.AppPrimary,
+          compoundBrandForeground1Hover: ramp.AppShade10,
+          compoundBrandForeground1Pressed: ramp.AppShade20,
+          brandForeground1: ramp.AppPrimary,
+          brandForeground2: ramp.AppShade10,
+          brandBackground: ramp.AppPrimary,
+          brandBackgroundHover: ramp.AppShade10,
+          brandBackgroundPressed: ramp.AppShade30,
+          compoundBrandBackground1: ramp.AppPrimary,
+          compoundBrandBackground1Hover: ramp.AppShade10,
+          compoundBrandBackground1Pressed: ramp.AppShade20,
+          brandStroke1: ramp.AppPrimary,
+          brandStroke2: ramp.AppTint40,
+          compoundBrandStroke1: ramp.AppPrimary,
+          compoundBrandStroke1Hover: ramp.AppShade10,
+          compoundBrandStroke1Pressed: ramp.AppShade20,
```

Comment:
For my own knowledge, Both the left hand side and the right hadn't side are technically alias tokens, yes?

Reply:
Yeah, I think technically that's true. The right side will change value based on which app you're using, so it would be considered an alias token.